packages/auth/addon/templates/components/spree-auth-orders.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="row">
+    <h3>My Orders</h3>
+    <table class="small-12 columns">
+        <thead class="active">
+        <tr>
+            <th class="order-number">Number</th>
+            <th class="order-date">Date</th>
+            <th class="order-status">Status</th>
+            <th class="order-payment-state">Payment State</th>
+            <th class="order-shipment-state">Shipment State</th>
+            <th class="order-total">Total</th>
+        </tr>
+        </thead>
+        <tbody>
+        {{#each order in orders}}
+          {{#if order.isComplete}}
+              <tr>
+                  <td class="order-number">{{#link-to 'spree.orders.show' order.number}}{{order.number}}{{/link-to}}</td>
```

Comment:
NICE!

I think if we just move this layout onto the account template, everything will be a bit simpler :+1: